pkg/kubelet/dockertools/manager.go

```diff
@@ -924,7 +929,7 @@ func (dm *DockerManager) podInfraContainerChanged(pod *api.Pod, podInfraContaine
 		networkMode = dockerPodInfraContainer.HostConfig.NetworkMode
 	}
 	if pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.HostNetwork {
```

Comment:
Can we just do something like:

func useHostNetwork(pod *api.Pod) bool {
	return pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.HostNetwork)
}

and then call useHostNetwork() from all the places that do the HostNetwork check?